controllers/UserController.js

```diff
@@ -0,0 +1,44 @@
+const apiResponse = require("../helpers/apiResponse");
+const UserModel = require("../models/UserModel");
+const { validationResult } = require("express-validator");
+const auth = require("../middlewares/jwt");
+
+exports.getUser = [
+  auth,
+  (req, res) => {
+    try {
+      const user = req.user;
+
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
+        return apiResponse.validationErrorWithData(
+          res,
+          "Validation Error.",
+          errors.array()
+        );
+      }
+
+      UserModel.findOne(
+        { _id: user.id },
+        {
+          name: 1,
+          email: 1,
+          userType: 1,
+          isOnboardingCompleted: 1,
+          recruiter: 1,
+        }
+      )
+        .populate({ path: "recruiter", model: "Recruiter" })
+        .populate({ path: "jobseeker", model: "JobSeeker" })
+        .exec((error, user) => {
+          if (error) {
+            return apiResponse.ErrorResponse(res, error);
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 tabs but found 12 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
						return apiResponse.ErrorResponse(res, error);
```